chromium_src/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_BOOKMARKS_BOOKMARK_MENU_DELEGATE_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_BOOKMARKS_BOOKMARK_MENU_DELEGATE_H_
+
+#include "brave/browser/ui/toolbar/brave_bookmark_context_menu_controller.h"
+
+// Adjust maximum menu width to fit German localization.
+#define GetMaxWidthForMenu(...)                 \
+  GetMaxWidthForMenu_ChromiumImpl(__VA_ARGS__); \
```

Comment:
Maybe append `UnUsed()` is better in this case as it's not used?
Just a suggestion. It's up to you.
```suggestion
  GetMaxWidthForMenu_UnUsed(__VA_ARGS__); \
```

Reply:
Good point! Done in [e844d26](https://github.com/brave/brave-core/commit/e844d269d4d23978a06b62dd155fa57d3bf4dd69)